cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -100,7 +100,7 @@ def handle_grading_policy_changed(sender, **kwargs):
         'event_transaction_id': unicode(get_event_transaction_id()),
         'event_transaction_type': unicode(get_event_transaction_type()),
     }
-    result = compute_all_grades_for_course.apply_async(kwargs=kwargs)
+    result = compute_all_grades_for_course.apply_async(kwargs=kwargs, countdown=1800)
```

Comment:
Let's define a constant for this, like `GRADING_POLICY_COUNTDOWN_SECONDS` or something.